### DISCUSSION TITLE
Welcome to obsidian-zhongwen-block Discussions!

### DISCUSSION THREAD

**Discussion by 0918nobita** [0918nobita/obsidian-zhongwen-block#89] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zlr2024** [comment]
PDF和word文件怎么转换成markdown，写进obsidian，之前有个PDF to markdown的插件，但是自带的没有了，手动的装不上。请问有什么好的解决方法嘛？

**Comment by serjofrancisco** [comment]
How do i insert line breaks? 
this:
![image](https://github.com/user-attachments/assets/f65b9cda-52ce-4c70-8bc2-65a699ec38cd)
always come as this: 
![image](https://github.com/user-attachments/assets/ee4282eb-6d2b-4e23-9b69-63088f60a6a0)

I'm doing something wrong?
